security/service-policy.md

```diff
@@ -71,6 +72,49 @@ spec:
 > **Note**: This policy also enacts a default-deny behavior for all pods, so make sure any other required application traffic is allowed by a policy.
 {: .alert .alert-info}
 
+### Allow access from a specified service
+
+In the following example, ingress traffic is allowed from `my-service` service in the `my-app` namespace for all pods in the namespace `my-other-app`.
```

Comment:
we should change the example names to be something at least slightly more realistic sounding - eg., "Allow traffic from frontend-service to backend-service"